internal/tools/rootfs/main.go

```diff
@@ -0,0 +1,111 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// --------------------------------------------------------------------------------------------
+
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/Microsoft/hcsshim/ext4/tar2ext4"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/daemon"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
+)
+
+func main() {
+	image := flag.String("i", "", "image")
+	tag := flag.String("t", "latest", "tag")
+	destination := flag.String("d", "local", "destination")
+
+	flag.Parse()
+	if flag.NArg() != 0 || len(*image) == 0 {
+		flag.Usage()
+		os.Exit(1)
+	}
+
+	layers, err := getRootfsLayerHashes(*image, *tag, *destination)
+
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+
+	for i, layer := range layers {
+		fmt.Printf("[%d]: %s\n", i, layer)
+	}
+}
+
+func getRootfsLayerHashes(image_name string, image_tag string, destination string) ([]string, error) {
```

Comment:
Go tends to use camel case for variable names. 
```suggestion
func getRootfsLayerHashes(imageName string, imageTag string, destination string) ([]string, error) {
```